lib/static/new-ui/components/MetaInfo/index.tsx

```diff
@@ -109,6 +110,23 @@ function MetaInfoInternal(props: MetaInfoInternalProps): ReactNode {
         });
     }
 
+    const shouldAddSkipReason = Boolean(!metaInfoItemsWithResolvedUrls.find(item => item.label === 'muteReason'));
+    if (result.skipReason && shouldAddSkipReason) {
+        const reason = Parser(result.skipReason);
+        if (typeof reason === 'string') {
+            metaInfoItemsWithResolvedUrls.push({
+                label: 'skipReason',
+                content: reason
+            });
+        } else if (!Array.isArray(reason) && typeof reason.props.children === 'string' && typeof reason.props.href === 'string') {
```

Comment:
Reason can be an array?

Reply:
This is no longer here, got rid of parsing html entirely.